include/UIKit/UIApplication.h

```diff
@@ -262,15 +241,15 @@ UIKIT_EXPORT_CLASS
 @property (getter=isProximitySensingEnabled, nonatomic) BOOL proximitySensingEnabled STUB_PROPERTY;
 @property (nonatomic) BOOL applicationSupportsShakeToEdit STUB_PROPERTY;
 @property (nonatomic) NSInteger applicationIconBadgeNumber;
-@property (nonatomic) UIInterfaceOrientation statusBarOrientation;
+@property (nonatomic) UIInterfaceOrientation statusBarOrientation STUB_PROPERTY;
 @property (nonatomic, assign) id<UIApplicationDelegate> delegate;
-@property (nonatomic, copy) NSString* preferredContentSizeCategory;
-@property (nonatomic, getter=isStatusBarHidden) BOOL statusBarHidden;
-@property (nonatomic, readonly) CGRect statusBarFrame;
+@property (nonatomic, copy) NSString* preferredContentSizeCategory STUB_PROPERTY;
+@property (nonatomic, getter=isStatusBarHidden) BOOL statusBarHidden STUB_PROPERTY;
+@property (nonatomic, readonly) CGRect statusBarFrame STUB_PROPERTY;
```

Comment:
oh nice.  too bad none of UIKit knew about this :(

---
In reply to: [110456645](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r110456645) [](ancestors = 110456645)